src/journey-maps/angular/services/map/map-routes.service.ts

```diff
@@ -54,12 +54,7 @@ export class SbbMapRoutesService {
     this._mapRouteService.updateRoute(
       map,
       mapSelectionEventService,
-      toFeatureCollection(
-        routes.reduce(
-          (accumulatedFeatures, next) => accumulatedFeatures.concat(next.features as any),
-          []
-        )
-      )
+      toFeatureCollection(routes.flatMap((r) => r.features))
```

Comment:
directly fixes a comment that was unintentionally deleted (IntelliJ reformat) in another PR: https://github.com/sbb-design-systems/sbb-angular/pull/1789/files#diff-c95089699f3e93d1afd29196dd53830d3d40028f7798a82e20ab37bc2cf09d9eR57